omniport/core/open_auth/serializers/application.py

```diff
@@ -135,10 +135,18 @@ def validate_redirect_uris(self, value):
         """
 
         validator = URLValidator()
+        schemes = ['http', 'https', 'ftp', 'ftps']
 
         redirect_urls = value.split(' ')
         for url in redirect_urls:
-            validator(url)
+            if '://' not in url:
+                url = 'http://' + url
+            url_scheme = url.split('://')[0].lower()
```

Comment:
Attaching `http://` to the URL just prior to splitting at `://` will give you `'http'`. Isn't there a more elegant way to do this?

Reply:
Like I have done this in case no '://'(delimiter) and scheme is provided otherwise the custom URL is still validated with any scheme, (with http, https, ftp and ftps) as it happens in URL Validator.

Reply:
Should we validate the URLs without mentioning any protocol? Or should we consider writing a custom URL validator all by scratch as suggested by @algomaster99 this is more of a hack?

Reply:
I have made changes for validating the url without scheme.